docs/tutorials/fix_scenarios.md

```diff
@@ -258,22 +271,40 @@ Enable services with: pro enable <service>
 Technical support level: essential
 { apt update && apt install --only-upgrade -y curl libcurl3-gnutls }
 ✔ USN-5079-2 is resolved.
+
+Found related USNs:
+- USN-5079-1
+
+Fixing related USNs:
+- USN-5079-1
+No affected source packages are installed.
+
+✔ USN-5079-1 does not affect your system.
+
+Summary:
+✔ USN-5079-2 [requested] is resolved.
+✔ USN-5079-1 [related] does not affect your system.
 ```
 
-We can see that that the attach command was successful, which can be verified
+Note that we can see that this command also fixed related USNs **USN-5079-1**.
+If you want to learn more about related USNs, please that a look into this [document](../references/what_are_related_usns)
```

Comment:
```suggestion
If you want to learn more about related USNs, refer to [our explanation guide](../references/what_are_related_usns).
```

Reply:
This link may change if we combine all three pages - we can point to the header so it jumps to the right part of the page.